docs/scos/dev/developer-getting-started-guide.md

```diff
@@ -160,3 +197,52 @@ Get to know the parts of the Spryker Development Virtual Machine with which we s
 * What is the Spryker DevVM (Development Virtual Machine) and why do we need it?
 * Main Structure
 * Technology Stack: Linux distribution, PHP, Postgres, MySQL, ES, Redis, Queue, Jenkins-->
+
+## 6. CI configuration
```

Comment:
What are the links below for? We need to provide some introduction to specify why we are referring to those links